synthtool/gcp/templates/node_library/.kokoro/release/publish.cfg

```diff
@@ -30,7 +30,7 @@ build_file: "{{ metadata['repository_name'] }}/.kokoro/trampoline_v2.sh"
 # Configure the docker image for kokoro-trampoline.
 env_vars: {
     key: "TRAMPOLINE_IMAGE"
-    value: "gcr.io/cloud-devrel-kokoro-resources/node:14-user"
+    value: "gcr.io/cloud-devrel-kokoro-resources/node:18-user"
```

Comment:
I'm sure this should exist, because: https://github.com/googleapis/testing-infra-docker/blob/main/node/18-user/Dockerfile

but I also don't see it here: https://pantheon.corp.google.com/artifacts/docker/cloud-devrel-kokoro-resources/us/gcr.io?inv=1&invt=AbcqFw&project=cloud-devrel-kokoro-resources

would you mind confirming this image is usable?